packages/core/src/transaction/Clause.ts

```
@@ -267,12 +268,17 @@ class Clause implements TransactionClause {
         token: Token
     ): Clause {
         if (token.value >= 0) {
+            const tokenValueWei = Units.convertUnits(
+                FixedPointNumber.of(token.value),
+                token.units,
+                Units.wei
+            );
             return this.callFunction(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)